config/stage_1_style_tests/check_lint.py

```diff
@@ -34,6 +37,8 @@ def check_lint_on_paths(paths: list[Path], path_to_config: Path,
         "--rcfile",
         str(path_to_config)
     ]
+    if ignore_tests:
+        lint_args.extend(["--ignore", "tests"])
```

Comment:
Why without lab name?

Reply:
According to pylint documentation it is a basename, not path